examples/wakaama/Makefile

```diff
@@ -0,0 +1,86 @@
+# name of your application
+APPLICATION = wakaama
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# TinyDTLS only has support for 32-bit architectures ATM
+BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno chronos \
+                   msb-430 msb-430h telosb waspmote-pro wsn430-v1_3b wsn430-v1_4 \
+                   z1
+
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon b-l072z-lrwan1 bluepill calliope-mini \
+                             cc2650-launchpad cc2650stk maple-mini \
+                             microbit nrf51dongle nrf6310 nucleo32-f031 \
+                             nucleo32-f042 nucleo32-f303 nucleo32-l031 nucleo-f030 \
+                             nucleo-f070 nucleo-f072 nucleo-f103 nucleo-f302 nucleo-f334 \
+                             nucleo-l053 nucleo-l073 opencm904 \
+                             spark-core stm32f0discovery yunjia-nrf51822
+
+# Include packages that pull up and auto-init the link layer.
+# NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+# Specify the mandatory networking modules for IPv6 and sUDP
+USEMODULE += gnrc_ipv6_router_default
+USEMODULE += gnrc_udp
+# Add a routing protocol
+USEMODULE += gnrc_rpl
+# This application dumps received packets to STDIO using the pktdump module
+USEMODULE += gnrc_pktdump
+# Additional networking modules that can be dropped if not needed
+USEMODULE += gnrc_icmpv6_echo
+# Add also the shell, some shell commands
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+#TinyDTLs (crypto.c) made use of pthread
+ifneq ($(BOARD),native)
+   USEMODULE += pthread
+endif
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+
+# NOTE: Add the package for TinyDTLS
+USEPKG += tinydtls
+
+# NOTE: Those are taken from TinyDTLS. As the original Makefiles are
+#      overwitten is a good idea to preserve them here.
+CFLAGS += -DDTLSv12 -DWITH_SHA256
+
+# NOTE: This adds support for TLS_PSK_WITH_AES_128_CCM_8
+CFLAGS += -DDTLS_PSK
+
+# NOTE: This adds support for TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8
+CFLAGS += -DDTLS_ECC
```

Comment:
After commit bb8ca70 and because you were testing only the PSK mode, it's a good idea to disable this line (or remove it).

You can use `git commit --fixup==bb8ca70d4d33212d3a7b37b57fc1c25a2f38160f` for this change. Though is not relevant as then both commitess will be squashed.

Reply:
Since, as you said, all that will be squashed anyway, I just snuck that into the latest commit, since it's not that dramatic of a change.